java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1474,7 +1478,21 @@ public static Builder builder(DType type, int rows) {
    * @return the builder to use.
    */
   public static Builder builder(DType type, TimeUnit tsTimeUnit, int rows) {
-    return new Builder(type, tsTimeUnit, rows);
+    return new Builder(type, tsTimeUnit, rows, 0);
+  }
+
+  /**
+   * Create a new Builder to hold the specified number of rows and with enough space to hold the
+   * given amount of string data. Be sure to close the builder when done with it. Please try to
```

Comment:
by closing the builder do you mean call builder.build()?

Reply:
Yes it is exactly the same as with the other comments on builder.